src/Microsoft.VisualStudio.LanguageServices.Razor/Editor/RazorDirectiveCompletionProvider.cs

```diff
@@ -147,12 +147,19 @@ internal virtual IEnumerable<CompletionItem> GetCompletionItems(RazorSyntaxTree
             return completionItems;
         }
 
-        private bool AtDirectiveCompletionPoint(RazorSyntaxTree syntaxTree, CompletionContext context)
+        // Internal for testing
+        internal bool AtDirectiveCompletionPoint(RazorSyntaxTree syntaxTree, CompletionContext context)
```

Comment:
Can you log a bug also to fix the underlying problem here? I think this is the right fix for VS15.5, but we need to ultimately make this work.

Reply:
Interesting, are you thinking of a fix where the directive completion provider awaits the completion of a parse in order to determine completion?